### PR TITLE
DOCS-422 update Cluster-configuration-settings.md

### PR DESCRIPTION
**Pull request by ebarcott-aerospike**

### CODE REVIEW

File: docs/Cluster-configuration-settings.md [hunk 1]
```diff
@@ -3,73 +3,52 @@ title: Cluster Configuration Settings
 description: Cluster Configuration Settings
 ---
 
-## Custom Resource Definition and Custom Resource
+Aerospike cluster configuration settings are found in the Aerospike cluster Custom Resource (CR). You can edit this file to make changes to your Aerospike cluster.
 
-The Operator [CRD](https://github.com/aerospike/aerospike-kubernetes-operator/tree/2.0.0-rc1/deploy/crds/aerospike.com_aerospikeclusters_crd.yaml) specifies the CR that the operator uses. The Aerospike cluster Custom Resource (CR) based on this CRD drives the deployment and management of Aerospike clusters. To create and deploy an Aerospike cluster, create a CR yaml file.
+The Operator Custom Resource Definition (CRD) specifies the CR the Operator uses to manage the cluster.
```
Comment (arrowplum) [inline docs/Cluster-configuration-settings.md:8]: Hmm another "why aren't we linking out" question.

File: docs/Cluster-configuration-settings.md [hunk 2]
```diff
@@ -3,73 +3,52 @@ title: Cluster Configuration Settings
 description: Cluster Configuration Settings
 ---
 
-## Custom Resource Definition and Custom Resource
+Aerospike cluster configuration settings are found in the Aerospike cluster Custom Resource (CR). You can edit this file to make changes to your Aerospike cluster.
```
Comment (carlyorke99) [inline docs/Cluster-configuration-settings.md:6]:
```suggestion
Aerospike cluster configuration settings are in the Aerospike cluster Custom Resource (CR). You can edit this file to make changes to your Aerospike cluster.
```

File: docs/Cluster-configuration-settings.md [hunk 3]
```diff
@@ -219,14 +187,12 @@ Configures the aerospike-server container created by operator.
 
 ## Storage
 
-The storage section configures persistent volumes devices to provision and attach to the containers.
+The storage section configures persistent volume devices to provision and attach to the containers.
 
-This section is required by default for persisting the Aerospike [work directory](https://docs.aerospike.com/docs/configuration/index.md?show-removed=1#work-directory). The working directory should be stored on persistent storage to ensure pod restarts do not reset Aerospike server metadata files.
+This section is required by default for persisting the Aerospike work directory. The work directory should be stored on persistent storage to ensure pod restarts do not reset Aerospike server metadata files.
```
Comment (carlyorke99) [inline docs/Cluster-configuration-settings.md:192]:
```suggestion
This section is required by default for persisting the Aerospike work directory. Store the work directory on persistent storage to ensure pod restarts do not reset Aerospike server metadata files.
```

File: docs/Cluster-configuration-settings.md [hunk 4]
```diff
@@ -247,13 +211,13 @@ The fields are
 For filesystem volumes, initMethod can be `none` or `deleteFiles`.
 For block volumes, initMethod can be `none`, `dd` or `blkdiscard`.
 
-Note: `blkdiscard` will only work for devices that support [TRIM](https://en.wikipedia.org/wiki/Trim_%28computing%29). For AWS please refer to the [storage volumes guide](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html#instance-store-volumes) to check TRIM support. If trim is not supported please use the slower `dd` in the case your devices need initialization. For other devices please verify the support for TRIM command.
+:::note
+`blkdiscard` will only work for devices which support [TRIM](https://en.wikipedia.org/wiki/Trim_%28computing%29). For AWS please refer to the [storage volumes guide](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html#instance-store-volumes) to check TRIM support. If TRIM is not supported please use the slower `dd` if your devices need initialization. For other devices please verify the support for TRIM command.
```
Comment (carlyorke99) [inline docs/Cluster-configuration-settings.md:215]:
```suggestion
`blkdiscard` only works for devices which support [TRIM](https://en.wikipedia.org/wiki/Trim_%28computing%29). For AWS please refer to the [storage volumes guide](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html#instance-store-volumes) to check TRIM support. If TRIM is not supported please use the slower `dd` to initialize your devices. For other devices please verify the support for TRIM command.
```

File: docs/Cluster-configuration-settings.md [hunk 5]
```diff
@@ -268,21 +232,21 @@ The fields are
 For filesystem volumes, initMethod can be `none` or `deleteFiles`.
 For block volumes, initMethod can be `none`, `dd` or `blkdiscard`.
 
-Note: `blkdiscard` will only work for devices that support [TRIM](https://en.wikipedia.org/wiki/Trim_%28computing%29). For AWS please refer to the [storage volumes guide](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html#instance-store-volumes) to check TRIM support. If trim is not supported please use the slower `dd` in the case your devices need initialization. For other devices please verify the support for TRIM command.
+:::note
+`blkdiscard` will only work for devices that support [TRIM](https://en.wikipedia.org/wiki/Trim_%28computing%29). For AWS please refer to the [storage volumes guide](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html#instance-store-volumes) to check TRIM support. If TRIM is not supported please use the slower `dd` if your devices need initialization. For other devices please verify the support for TRIM command.
```
Comment (carlyorke99) [inline docs/Cluster-configuration-settings.md:236]:
```suggestion
`blkdiscard` only works for devices that support [TRIM](https://en.wikipedia.org/wiki/Trim_%28computing%29). For AWS please refer to the [storage volumes guide](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html#instance-store-volumes) to check TRIM support. If TRIM is not supported please use the slower `dd` to initialize your devices. For other devices please verify the support for TRIM command.
```

File: docs/Cluster-configuration-settings.md [hunk 6]
```diff
@@ -338,10 +305,9 @@ Example
 ```
 
 #### ConfigMap
-A Kubernetes [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/) can be mounter as  a volume.
-A config map hold non-confidential data which could be configuration four applications running in your custom init-containers or sidecars.
 
-Example
+A Kubernetes ConfigMap can be mounted as  a volume. ConfigMaps hold non-confidential data like configuration for applications running in your custom init containers or sidecars.
```
Comment (carlyorke99) [inline docs/Cluster-configuration-settings.md:309]:
```suggestion
A Kubernetes ConfigMap can be mounted as a volume. ConfigMaps hold non-confidential data like configuration for applications running in your custom init containers or sidecars.
```

File: docs/Cluster-configuration-settings.md [hunk 7]
```diff
@@ -432,12 +394,11 @@ Configures the name of the secret to use and the mount path to mount the secret
...
 ## Aerospike Config
 
-The YAML form of Aerospike server configuration.
-See [Aerospike Configuration](Aerospike-configuration-mapping.md) for details.
+The YAML form of Aerospike server configuration. See [Aerospike Configuration](Aerospike-configuration-mapping.md) for details.
 
 ## Rack Config
 
-Configures the operator to deploy rack aware Aerospike cluster. Pods will be deployed in given racks based on the given configuration.
+Configures the operator to deploy a rack-aware Aerospike cluster. Pods will be deployed in given racks based on the given configuration.
```
Comment (carlyorke99) [inline docs/Cluster-configuration-settings.md:401]:
```suggestion
Configures the operator to deploy a rack-aware Aerospike cluster. Pods are deployed in given racks based on the given configuration.
```

**Review by carlyorke99 — APPROVED** [review]

**Pull request merged**